drivers/espi/espi_taf_npcx.c

```diff
@@ -620,6 +689,20 @@ static int espi_taf_npcx_init(const struct device *dev)
 		  config->max_rd_sz);
 	inst->FLASHBASE = config->mapped_addr;
 
+#if DT_NODE_HAS_STATUS_OKAY(NPCX_TAF_SEC_FLASH_NODE)
+	if (IS_ENABLED(CONFIG_FLASH_NPCX_FIU_SUPP_LOW_DEV_SWAP)) {
+		npcx_espi_taf_data.low_dev_ptr = spi_dev1;
+		npcx_espi_taf_data.high_dev_ptr = spi_dev;
+		npcx_espi_taf_data.low_dev_size =
+			MB(1) << DT_ENUM_IDX(NPCX_TAF_SEC_FLASH_NODE, spi_dev_size);
```

Comment:
this could probably use a define at the top of the file, then the line would not wrap

no need for guards if it's not used it just won't get evaluated

Reply:
Have added the definition for the low_dev_size parameter to let the line would not wrap. 

In this part, the init will check whether the status of the second flash node is okay and whether it needs to reverse the flash device node. 

Thus, I am confused about the statement 'no need for guards if it's not used it just won't get evaluated.' Could you please elaborate further to clarify it? It would help to better understand this comment. Thank you.

Reply:
ah nothing was just referring to the fact that this code is in an include guard